index.html

```diff
@@ -32,37 +31,222 @@ <h4>My Logo</h4>
+                                See Project
+                            </a>
+                        </div>
+                    </div>
+                </article>
+            </section>
+            <section id="about-me-more" class="about-me-more">
                 <div>
                     <div>
-                        <h1>
-                            I'm William
-                            Glad to see you!
-                        </h1>
+                        <h2>
+                            About <br />
+                            Myself
+                        </h2>
                     </div>
                     <div class="description">
-                        <p>I'm a software developer! I can help you build a product , feature or website Look through
-                            some
-                            of my
-                            work and
-                            experience! If you like what you see and have a project you need coded, don’t hestiate to
-                            contact
-                            me.</p>
+                        <p>Hello I'm a software developer! I can help you build a product , feature or website Look through some of my work and
+                        experience! If you like what you see and have a project you need coded, don't hestiate to contact me.</p>
                     </div>
                     <div class="connect">
                         <h5>LET'S CONNECT</h5>
                         <ul>
-                            <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
-                            <li><a href="#"><img src="assets/img/linkedin.png" alt="linkedin" /></a></li>
-                            <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
                             <li><a href="#"><img src="assets/img/github.png" alt="github" /></a></li>
+                            <li><a href="#"><img src="assets/img/linkedin.png" alt="linkedin" /></a></li>
                             <li><a href="#"><img src="assets/img/angies.png" alt="angies" /></a></li>
+                            <li><a href="#"><img src="assets/img/twitter.png" alt="twitter" /></a></li>
+                            <li><a href="#"><img src="assets/img/medium.png" alt="medium" /></a></li>
                         </ul>
                     </div>
+                    <div class="read-more">
+                        <a href="#">
+                            Get my resume
+                        </a>
+                    </div>
+                </div>
+                <div>
+                    <ul>
+                        <li class="categories open">
+                            <a href="#"><span>Languages</span><span><img src="./assets/img/arrow-down.png"
+                                        alt="more" /></span></a>
+                            <div class="item-list">
+                                <div class="item">
```

Comment:
I think your about section looks a little bit different from the design, kindly see the images below🤗
Your page
<img width="268" alt="your_about" src="https://user-images.githubusercontent.com/64643995/212485729-7ee89147-4026-45f3-ae2d-b77ba2e77969.png">

the design
<img width="224" alt="design_about" src="https://user-images.githubusercontent.com/64643995/212485740-0b94a69b-99b1-482c-9d0b-9f2199d59b52.png">

kindly make the appropriate adjustments, as it is a requirement that we stick to the design for this milestone🤗